lib/form_data.js

```diff
@@ -159,7 +159,7 @@ FormData.prototype._trackLength = function(header, value, options) {
 };
 
 // TODO: Use request's response mime-type
-FormData.prototype._multiPartHeader = function(field, value, options) {
+FormData.prototype._multiPartHeader = function(field, value, options,encodeFieldNamesWithoutQuotes) {
```

Comment:
I would recommend to move this parameter to the options, since it may be passed here anyway.